config/nova-command-runner.php

```diff
@@ -54,6 +54,9 @@
 
         // Queueing commands with custom connection and queue
         'Clear Cache' => [ 'run' => 'cache:clear --should-queue --cr-connection=database --cr-queue=high', 'type' => 'danger', 'group' => 'Cache' ],
+
+        // Queueing long-running commands causing big output with custom timeout limit and output size
+        'Update Prices' => [ 'run' => 'prices:update', 'type' => 'primary', 'group' => 'Sync', 'timeout' => 120, 'output_size' => 3 ],
```

Comment:
I think this is here by mistake.

Reply:
Good catch, thanks, have moved this to readme.